rails/how-to/feature_test_javascript_in_a_rails_app.md

```diff
@@ -17,11 +17,11 @@ end
 ```
 
 When writing a spec, you must set the `:js` flag for that test to make use of
-capybara-webkit. For example, in `spec/features/user_signs_in_spec.rb`:
+capybara-webkit. For example, in `spec/system/user_signs_in_spec.rb`:
 
 ```ruby
-feature "Authentication", :js do
-  scenario "A user signing in" do
+describe "Authentication", :js do
+  it "A user signing in" do
```

Comment:
I think if we want to use `it`, we should update the description as well "it a user signing it" doesn't read well. Some options:

```suggestion
  it "allows a user to sign in" do
```

```suggestion
  it "signs in a user" do
```